oak_attestation_verification/src/claims.rs

```diff
@@ -23,8 +23,9 @@ extern crate alloc;
 
 use alloc::{collections::BTreeMap, string::String, vec::Vec};
 
-use anyhow::Context;
-use serde::{Deserialize, Serialize};
+use serde::Deserialize;
+#[cfg(feature = "serialize")]
```

Comment:
Probably not worth going back and changing this now, but perhaps we should just have a `std` feature to gate all the non-no-std stuff? I think that's what some external crates do. (but we can follow up, no need to interrupt this presubmit now)

Reply:
Acknowledged. I'll put together a separate PR.